helpers/test/test_lib_tasks_docker_release.py

```diff
@@ -0,0 +1,184 @@
+        cp -f poetry.lock.out ./devops/docker_build/poetry.lock
+        cp -f pip_list.txt ./devops/docker_build/pip_list.txt
+        docker image ls test-registry.com/test-image:local-$USER_NAME-1.0.0
+        """
+        # Check output.
+        self.assert_equal(
+            actual_cmds,
+            exp,
+            purify_text=True,
+            fuzzy_match=True,
+            remove_lead_trail_empty_lines=True,
+            dedent=True,
+        )
+
+    def test_docker_build_multi_arch(self) -> None:
+        """
+        Test building a local Docker image with multiple architectures.
+
+        This test verifies that the correct sequence of commands is
+        generated for building a local Docker image with multiple
+        architectures.
+        """
+        # Prepare inputs.
+        mock_ctx = httestlib._build_mock_context_returning_ok()
+        test_version = "1.0.0"
+        test_base_image = "test-registry.com/test-image"
+        test_multi_arch = "linux/amd64,linux/arm64"
+        # Call tested function.
+        hltadore.docker_build_local_image(
+            mock_ctx,
+            test_version,
+            cache=False,
+            base_image=test_base_image,
+            poetry_mode="update",
+            multi_arch=test_multi_arch,
+        )
+        actual_cmds = _extract_commands_from_call(self.mock_run.call_args_list)
+        actual_cmds = "\n".join(actual_cmds)
+        # The output is a list of strings, each representing a command.
+        exp = r"""
+        cp -f devops/docker_build/dockerignore.dev .dockerignore
+        docker buildx create \
+            --name multiarch_builder \
+            --driver docker-container \
+            --bootstrap \
+            && \
+            docker buildx use multiarch_builder
+        tar -czh . | DOCKER_BUILDKIT=0 \
+            time \
+            docker buildx build \
+            --no-cache \
+            --push \
+            --platform linux/amd64,linux/arm64 \
+            --build-arg AM_CONTAINER_VERSION=1.0.0 --build-arg INSTALL_DIND=True --build-arg POETRY_MODE=update --build-arg CLEAN_UP_INSTALLATION=True \
+            --tag test-registry.com/test-image:local-$USER_NAME-1.0.0 \
+            --file devops/docker_build/dev.Dockerfile \
+            -
+        docker pull test-registry.com/test-image:local-$USER_NAME-1.0.0
+        invoke docker_cmd --stage local --version 1.0.0 --cmd 'cp -f /install/poetry.lock.out /install/pip_list.txt .' --skip-pull
+        cp -f poetry.lock.out ./devops/docker_build/poetry.lock
+        cp -f pip_list.txt ./devops/docker_build/pip_list.txt
+        docker image ls test-registry.com/test-image:local-$USER_NAME-1.0.0
+        """
+        # Check output.
+        self.assert_equal(
+            actual_cmds,
+            exp,
+            purify_text=True,
+            fuzzy_match=True,
+            remove_lead_trail_empty_lines=True,
+            dedent=True,
```

Comment:
I think it’s redundant since fuzzy_match=True.
Feel free to check and remove it everywhere if it works.

Reply:
It's ok, even if redundant. We'll keep working on this so let's merge